astroquery/sdss/core.py

```diff
@@ -277,16 +317,83 @@ def query_region_async(self, coordinates, *, radius=2. * u.arcsec, timeout=TIMEO
             The result of the query as a `~astropy.table.Table` object.
 
         """
-        request_payload, files = self.query_crossid_async(coordinates=coordinates,
-                                                          radius=radius, fields=fields,
-                                                          photoobj_fields=photoobj_fields,
-                                                          specobj_fields=specobj_fields,
-                                                          obj_names=obj_names,
-                                                          spectro=spectro,
-                                                          region=True,
-                                                          field_help=field_help,
-                                                          get_query_payload=True,
-                                                          data_release=data_release)
+        # Allow field_help requests to pass without requiring a radius or width.
+        if field_help and radius is None and width is None:
+            radius = 2.0 * u.arcsec
+
+        if radius is None and width is None:
+            raise ValueError("Either radius or width must be selected!")
+        if radius is not None and width is not None:
+            raise ValueError("Either radius or width must be selected!")
+
+        if radius is not None:
+            request_payload, files = self.query_crossid_async(coordinates=coordinates,
+                                                              radius=radius, fields=fields,
+                                                              photoobj_fields=photoobj_fields,
+                                                              specobj_fields=specobj_fields,
+                                                              obj_names=obj_names,
+                                                              spectro=spectro,
+                                                              region=True,
+                                                              field_help=field_help,
+                                                              get_query_payload=True,
+                                                              data_release=data_release)
+
+        if width is not None:
+            if isinstance(width, Angle):
+                width = width.to_value(u.degree)
+            else:
+                try:
+                    width = Angle(width).to_value(u.degree)
+                except ValueError:
+                    raise TypeError("width should be either Quantity or "
+                                    "convertible to float.")
```

Comment:
I think this achieves the same thing more tersely:
```suggestion
            width = u.Quantity(width, u.degree).value
```

Reply:
OK, that's a nice trick. Done. Although for the record, two possible types of exception are raised: `width = u.Quantity('foo', u.degree).value` raises `TypeError`, while `width = u.Quantity('2 things', u.degree).value` raises `ValueError`.